inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/pulsar/src/main/java/org/apache/inlong/sort/pulsar/PulsarTableOptions.java

```diff
@@ -89,7 +89,7 @@ private PulsarTableOptions() {
                                     .build());
 
     public static final ConfigOption<String> SOURCE_START_FROM_MESSAGE_ID =
```

Comment:
It is recommend to keep the meaning between the name of constant and the `ConfigOptions.key`

Reply:
the name changed already